notifier/rss.c

```diff
@@ -385,8 +385,8 @@ main(int  argc,				/* I - Number of command-line arguments */
 
   if (http)
   {
-    unlink(filename);
     httpClose(http);
+    unlink(filename);
```

Comment:
I don't see the point of this change.

Reply:
Consistency with the other block of code that does this.